config/crd/bases/apps.foundationdb.org_foundationdbrestores.yaml

```diff
@@ -1,166 +0,0 @@
----
```

Comment:
Why was this file removed?

Reply:
hmm, I didn't do this intentionally. let me fix it.